cmd/lvh/kernels/build.go

```diff
@@ -26,6 +26,8 @@ func buildCommand() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&arch, "arch", "", "target architecture to build the kernel, e.g. 'amd64' or 'arm64' (default to native architecture)")
+	cmd.Flags().StringVar(&dirName, dirNameCommand, "", dirNameHelp)
+	cmd.MarkFlagRequired(dirNameCommand)
```

Comment:
I think looking at this commit will make thing looks more clear https://github.com/cilium/little-vm-helper/pull/184/commits/04f465a6361912418681367cd936c11335ffead1, it's just me moving the persistent flag on `kernels` to all the necessary subcommands.

Reply:
OK, so I guess nothing is changing exactly (other than that it's now in several places to update). What I'm wondering though is.. if there's a default, why does the user need to specify it? Seems like it would be simpler UX to just not mark it as required.

Whichever way you think makes more sense.

Reply:
I don't really like this tbh, I would set the default to `_data` because we use that everywhere. However I think most of the commands will fail if the directory in question is not already created.

Reply:
That sounds solvable, sprinkle some mkdirs around any commands that do a write?

Reply:
https://github.com/cilium/little-vm-helper/issues/186